src/features/VisualNovel/pages/OverviewPage/OverviewPage.module.css

```diff
@@ -1,41 +1,6 @@
-.page {
-  display: grid;
-  grid-template-columns: minmax(var(--chakra-sizes-64), var(--chakra-sizes-72)) 1fr;
-  gap: var(--chakra-sizes-8);
-}
-
-.items {
-  display: flex;
-  margin-top: var(--chakra-sizes-3);
-  gap: var(--chakra-sizes-3);
-  flex-wrap: wrap;
-}
-
-.title {
-  text-transform: capitalize;
-}
-
-.staff {
-  --columns-count: auto-fit;
-
-  display: grid;
-  margin: var(--chakra-sizes-4) 0 var(--chakra-sizes-10);
-  grid-template-columns: repeat(var(--columns-count), minmax(var(--chakra-sizes-72), 1fr));
-  grid-gap: var(--chakra-sizes-4);
-  grid-auto-flow: row;
-}
-
-@media (min-width: calc(var(--chakra-sizes-72) * 3 + var(--chakra-sizes-4) * 2)) {
-  .grid {
-    --columns-count: 3;
+@media screen and (max-width: 30em) {

```

Comment:
Not sure if it's safe to use `em` for `max-width`

Reply:
[Chakra uses `em` for breakpoints](https://chakra-ui.com/docs/styled-system/features/responsive-styles)